src/main/java/com/sonymobile/tools/gerrit/gerritevents/dto/events/ChangeBasedEvent.java

```diff
@@ -75,49 +67,21 @@ public void setChange(Change change) {
         this.change = change;
     }
 
-/**
-     * The changed files in this patchset.
-     */
-    private List<String> files;
-
 
 
     /**
      * Queries gerrit for the files included in this patch set.
      *
      * @param gerritQueryHandler the query handler, responsible for the queries to gerrit.
      * @return a list of files that are part of this patch set.
+     *
+     * @deprecated use {@link Change#getFiles(GerritQueryHandler)} instead.
      */
+    @Deprecated
```

Comment:
What is the rational of moving this?

Reply:
To be able to query list of files outside of ChangeBasedEvent. i.e.
```java
for(Change change : change.getTopic().getChanges()) {
     List<String> files = change.getFiles(gerritQueryHandler);
    // do something with this files
}
```